src/components/HeroHome/index.js

```diff
@@ -1,22 +1,27 @@
 import React from 'react';
 
-const HeroHome = ({ imageUrl, imageTitle, preheader }) => (
-  <div
-    className="coa-HeroHome"
-    style={{
-      backgroundImage: `
-        linear-gradient(rgba(36, 11, 51, .3), rgba(36, 11, 51, .3)),
-        url(${imageUrl})
-      `,
-      backgroundSize: 'cover',
-      backgroundPosition: '50%',
-    }}
-    role="img"
-    aria-label={imageTitle}
-  >
-    <div className="container-fluid wrapper">
-      <span className="coa-HeroHome-preheader">{preheader}</span>
-      <h2 className="coa-HeroHome-header">Austin, TX</h2>
+const HeroHome = ({ imageFilename, imageTitle, preheader }) => (
+  <div className="coa-HeroHome-container">
+    <img srcset={`${imageFilename}.width-640.jpg 640w,
+                  ${imageFilename}.width-720.jpg 720w,
+                  ${imageFilename}.width-750.jpg 750w,
+                  ${imageFilename}.width-828.jpg 828w,
+                  ${imageFilename}.width-1080.jpg 1080w,
+                  ${imageFilename}.width-1440.jpg 1440w,
+                  ${imageFilename}.width-2160.jpg 2160w,`} 
+         src={`${imageFilename}.width-1080.jpg`} />
+    <div
+      className="coa-HeroHome"
+      role="img"
+      aria-label={imageTitle}
+      style={{
+        background: 'linear-gradient(rgba(36, 11, 51, .3), rgba(36, 11, 51, .3))',
```

Comment:
I know you didn't originally create these `rgba` values but maybe they should also store them in the variables Sass file?